posts/2019-04-11-Rust-1.34.0.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: post
+title: "Announcing Rust 1.34.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.34.0. Rust is a
+programming language that is empowering everyone to build reliable and
```

Comment:
"that is empowering" could be more concisely "that empowers"

Reply:
I think this is due to the website slogan:
> Empowering everyone to build reliable and efficient software. 

Maybe it should change, but probably not here and now =P

Reply:
Maybe the slogan could be in quotes or italics here?

Reply:
Not sure, it's worth considering for the next release, but we are quite close to deadline now.

Reply:
👌